src/components/timeline/Point.tsx

```diff
@@ -45,18 +45,20 @@ const Point = (props: PointPropsInternal) => {
     const tintColor = removeIconBackground ? Colors.$iconDefault : Colors.$iconDefaultLight;
     const iconSize = removeIconBackground ? undefined : ICON_SIZE;
     if (icon) {
-      return <Icon tintColor={tintColor} {...iconProps} size={iconSize} source={icon}/>;
+      return (
+        <Icon testID={`${testID}.icon`} tintColor={tintColor} {...iconProps} size={iconSize} source={icon}/>
```

Comment:
Can we have both icon and label at the same point?

Reply:
No we can't according to the guidelines it's Icon or Label not both of them together.
The point is small so Icon or Label sounds right.

Reply:
So why we need two different testIDs? What are we testing with these ids?

Reply:
I think it's better since those are different type of components.
Since there are only two cases I think it's ok to have testIDs, when using label you expect text, the driver is different and etc.